modules/game_list_module.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../classes/game'
+require_relative '../classes/label'
+require_relative '../classes/author'
+
+module GameLister
+  @games = []
+  @authors = []
+
+  def list_games
+    if @games.empty?
+      puts 'No games available'
+    else
+      puts 'List of games:'
+      @games.each_with_index do |game, index|
+        print "#{index + 1}. #{game.title} "
+        puts "published on #{game.publish_date} and is #{game.multiplayer ? 'multiplayer' : 'singleplayer'}"
+      end
+    end
+    puts "\n"
+  end
+
+  def add_game
+    print 'Please enter the publish date of the book YYYY-MM-DD:   '
+    publish_date = gets.chomp
+    print 'multiplayer status true or false:  '
+    multiplayer = gets.chomp
+    print 'Last played date YYYY-MM-DD:   '
+    last_played_at = gets.chomp
+    print "Author's First name: "
+    first_name = gets.chomp
```

Comment:
- [ ] @Last_played_at should be a date variable
there seems to be an error in the can_be_archived? method, you can't compare @@last_played_at a string variable with (Date.today - (2 * 365)) 
![error](https://github.com/Leeoasis/Ruby-catalogue/assets/59117151/20fc95a0-1180-4a7c-9aca-062678184daa)